features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
+
+  Rule: properties die onderdeel uitmaken van een groep, maar wel een unieke propertynaam hebben mogen met alleen de propertynaam worden opgenomen in fields
+    Scenario: opvragen veld met fields door opgeven hele pad naar het veld
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                      |
+      | voornamen                 | Johannnes Geurt |
+      | geslachtsnaam             | Janssen         |
+      | aanduidingNaamgebruik     | E               |
+      En de persoon heeft de volgende 'geboorte' gegevens
+      | naam                                      | waarde   |
+      | datum.type                                | Datum    |
+      | datum.datum (01.03.10)                    | 19860401 |
+      | land.code                                 | 6030     |
+      | plaats.code                               | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                             |
+      | type                | RaadpleegMetBurgerservicenummer                    |
+      | burgerservicenummer | 999994086                                          |
+      | fields              | geboorte.plaats.code                               |
+      Dan heeft de persoon met burgerservicenummer '999994086' de volgende 'geboorte' gegevens
+      | naam                 | waarde       |
+      | geboorte.plaats.code | 0518         |
+      En heeft de persoon GEEN andere properties
+
+    Scenario: opvragen veld met fields door opgeven laatste deel van het pad naar het veld
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                      |
+      | voornamen                 | Johannnes Geurt |
+      | geslachtsnaam             | Janssen         |
+      | aanduidingNaamgebruik     | E               |
+      En de persoon heeft de volgende 'geboorte' gegevens
+      | naam                                      | waarde   |
+      | datum (01.03.10)                          | 19860401 |
+      | land                                      | 6030     |
+      | plaats                                    | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                             |
+      | type                | RaadpleegMetBurgerservicenummer                    |
+      | burgerservicenummer | 999994086                                          |
+      | fields              | plaats.code |
```

Comment:
Dit pad is niet uniek. De volgende property paden voldoen hieraan:
- geboorte.plaats.code
- overlijden.plaats.code
- kinderen.geboorte.plaats.code
- ouders.geboorte.plaats.code
- partners.geboorte.plaats.code
- partners.aangaanHuwelijkPartnerschap.plaats.code